Windows 11 inclusion

SamBH opened the pull request:
Update to Client Devices design to include Windows 11, including hardening guidance from the ACSC.

ABAC updates to follow in future PR.

SamBH committed:
Windows 11 inclusion

Update to Client Devices design to include Windows 11

mattshawoobe reviewed (changes requested):
-Line 20/21 Can we please include Windows 11 support requires MECM 2107 for Win11 21H2 and 2207 for 22H2 https://learn.microsoft.com/en-us/mem/configmgr/core/plan-design/configs/support-for-windows-11

-Line 251 Worth mentioning a note to say "there are MECM minimum requirements for 22H2 Windows 11" ?
- Line 647 you said channels earlier not rings, should be consistent.
- Line 126,129,133,131,132 should be just Windows hardening guidance as it's all just Windows later on

SamBH committed:
Updates based on review

Updates based on feedbacl:
- Included minimum MECM version for Windows 11 management.
- Standardised referencing to Windows hardening guide
- No update to "rings" - Intune still refers to update rings

SamBH reviewed (commented):
Updates made as per review feedback

mattshawoobe reviewed (approved):
Approved after updates
